Doc/library/dis.rst

```diff
@@ -793,6 +798,52 @@ All of the following opcodes use their arguments.
    .. versionadded:: 3.6
 
 
+.. opcode:: BUILD_TUPLE_UNPACK (count)
+
+   Pops *count* iterables from the stack, joins them in a single tuple,
+   and pushes the result.  This bytecode is used for implementing iterable
```

Comment:
Though not technically incorrect, using the sentence "This bytecode is used .." for every case is somewhat repetitive IMO.It might be better to use the same format the other opcodes in the docs use and go for:

Implements iterable unpacking for <type> displays ``<type_display>``